src/Client/Views/SidebarView.fs

```diff
@@ -14,27 +14,27 @@ type SidebarView =
         Html.div [
             prop.className "grow overflow-y-auto"
             prop.children [
-                match model.ProtocolState with
-                | {WidgetTypes = Routing.WidgetTypes.BuildingBlock } ->
-                    BuildingBlock.Core.addBuildingBlockComponent model dispatch
-
-                | {WidgetTypes = Routing.WidgetTypes.TermSearch } ->
-                    TermSearch.Main (model, dispatch)
+                if model.ProtocolState.IsProtocolSearch then
+                    Protocol.SearchContainer.Main model dispatch
```

Comment:
Move this if-else logic into the protocol view logic, do not handle this in sidebarview anymore

Reply:
fixed